### DISCUSSION TITLE
Lithium + Caffeine

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#86] [Drug Interactions]
There is some evidence that caffeine consumption may decrease lithium levels. Because of the narrow therapeutic window of lithium, people taking this medication and using caffeine need to use caution if they increase or decrease their caffeine consumption.

Frigerio, S., Strawbridge, R., & Young, A. H. (2021). The impact of caffeine consumption on clinical symptoms in patients with bipolar disorder: A systematic review. Bipolar Disorders, 23(3), 241–251. https://doi.org/10.1111/bdi.12990

**Comment by utaninja** [comment]
Adding on to this-

Kunitake Y, Mizoguchi Y, Sogawa R, Matsushima J, Kato TA, Kawashima T, Monji A. Effect of Excessive Coffee Consumption on the Clinical Course of a Patient With Bipolar Disorder: A Case Report and Literature Review. Clin Neuropharmacol. 2017 Jul/Aug;40(4):160-162. doi: 10.1097/WNF.0000000000000222. PMID: 28622205.
Mester R, Toren P, Mizrachi I, Wolmer L, Karni N, Weizman A. Caffeine withdrawal increases lithium blood levels. Biol Psychiatry. 1995 Mar 1;37(5):348-50. doi: 10.1016/0006-3223(94)00243-V. PMID: 7748990.

These all seem to be conclusive of the decrease of levels. Do you think a caution level would be appropriate here?

To clarify "use caution if they increase or decrease their caffeine consumption." are you saying if they take it consistently at the same amount it is not an issue? It's an issue only if they have more or less than usual? From my understanding, it seems individuals taking Lithium should not drink caffeine.

**Comment by utaninja** [comment]
Pushed to https://github.com/TripSit/drugs/issues/304
